virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -39,12 +39,15 @@ include::modules/virt-creating-vm.adoc[leveloffset=+1]
 :virtualmachine!:
 
 include::modules/virt-vm-storage-volume-types.adoc[leveloffset=+1]
+include::modules/virt-about-runstrategies-vms.adoc[leveloffset=+1]
+* See xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[Deploying machine health checks] for further details on deploying and enabling machine health checks.
+* See xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer-provisioned installation overview] for further details on installer-provisioned installation and usage.
```

Comment:
Again about the "infrastructure", not "installation". I won't point out any more instances, if you can get the entire PR for all instances.

Also, I'm not sure if the "and usage" is appropriate, especially after changing the wording.